tests/operations.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { test } from "./fixtures/lxd-test";
+import {
+  createInstance,
+  deleteInstance,
+  randomInstanceName,
+  visitAndStartInstance,
+  visitAndStopInstance,
+} from "./helpers/instances";
+import { assertTextVisible } from "./helpers/permissions";
+import {
+  checkNotificationExists,
+  dismissNotification,
+} from "./helpers/notification";
+// import { validateOperation } from "./helpers/operations";
+
+let instance = randomInstanceName();
+
+test("instance operations are recognised on the Operations page", async ({
+  browser,
+  browserName,
+}) => {
+  // Setup
+  instance = `${browserName}-${instance}`;
+  const page = await browser.newPage();
+  await createInstance(page, instance);
+
+  await page.goto("/ui/operations");
+
+  // validate create operation is in operation list
+
+  await page
+    .getByRole("row", { name: "Action" })
+    .filter({ hasText: "Starting Instance" })
+    .filter({ hasText: instance })
+    .getByText(instance);
+
+  // await validateOperation(page, "Starting Instance", instance)
+
+  // start instance and wait for the notification instance was started
+  await visitAndStartInstance(page, instance);
+  await checkNotificationExists(page);
+  await assertTextVisible(page, "Instance " + instance + " started");
+
+  // click on the “x operations” link in the status bar
+  await dismissNotification(page);
+
+  // validate the operation to start the instance is in the operation list
+  await page.goto("/ui/operations");
+  await page
```

Comment:
Any playwright function that returns a `Locator` object shouldn't be awaited. e.g. `getByText`. Instead, actions or assertions on the `Locator` object should be awaited. e.g. `await page.getByText(instance).click()` or `await expect(page.getByText(instance)).toBeVisible()`